tests/test_packages/tools_for_testing.py

```diff
@@ -0,0 +1,172 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2019 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+
+"""This module contains tools for testing the packages."""
+
+import os
+import shutil
+import subprocess  # nosec
+import sys
+import tempfile
+
+import pytest
+
+from aea.cli import cli
+
+from tests.common.click_testing import CliRunner
+from tests.conftest import CLI_LOG_OPTION
+
+
+class AeaTestCase:
+    """Test case for AEA end-to-end tests."""
+
+    @pytest.fixture(autouse=True)
+    def _start_oef_node(self, network_node):
+        """Start an oef node."""
+
+    @classmethod
+    def setup_class(cls):
+        """Set up the test class."""
+        cls.runner = CliRunner()
+        cls.cwd = os.getcwd()
+        cls.t = tempfile.mkdtemp()
+
+        # add packages folder
+        packages_src = os.path.join(cls.cwd, "packages")
+        packages_dst = os.path.join(cls.t, "packages")
+        shutil.copytree(packages_src, packages_dst)
+
+        os.chdir(cls.t)
+
+    @classmethod
+    def teardown_class(cls):
+        """Teardowm the test."""
+        os.chdir(cls.cwd)
+        try:
+            shutil.rmtree(cls.t)
+        except (OSError, IOError):
+            pass
+
+    @staticmethod
+    def disable_ledger_tx(vendor_name, item_type, item_name):
```

Comment:
This is in my opinion misplaced here as it is not generic. Also, as mentioned in a previous comment, should use `aea config set ...` to update configs

Reply:
So what you could do is create a wrapper to call the config set command.

Reply:
Done.